src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearingpart/AssignHearingPartToSessionAction.java

```diff
@@ -85,15 +86,15 @@ public void act() {
     }
 
     @Override
-    public FactMessage generateFactMessage() {
+    public List<FactMessage> generateFactMessages() {
         String msg = null;
         //try {
```

Comment:
comments

Reply:
This action is not used anywhere. In of the PR someone is commenting this out. I would left it for now as we have in plans refactor it